### ISSUE TITLE
How to feed data to the graph after using Graph Transform Tool

### ISSUE THREAD

**Issue by StephenGreat** [tensorflow/tensorflow#23611]
The demo shows that it can strip unused nodes by Graph Transform Tool .
But when setting 'inputs=Mul:0' , the input of the graph becomes an 'Operation:Mul'.
The problem is that the graph needs a tensor as the input not the operation.
How to feed image data to the graph after stripping unused nodes?

bazel build tensorflow/tools/graph_transforms:transform_graph
bazel-bin/tensorflow/tools/graph_transforms/transform_graph \
--in_graph=tensorflow_inception_graph.pb \
--out_graph=optimized_inception_graph.pb \
--inputs='Mul:0' \
--outputs='softmax:0' \
--transforms='
strip_unused_nodes(type=float, shape="1,299,299,3")
remove_nodes(op=Identity, op=CheckNumerics)
fold_old_batch_norms
'

**Comment by harshini-gadige** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged//tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
